SPECS/setools/setools.spec

```diff
@@ -0,0 +1,182 @@
+%global selinux_ver 2.9-1
+%global __python3	/usr/bin/python3
+%define python3_sitearch %(python3 -c "from distutils.sysconfig import get_python_lib; import sys; sys.stdout.write(get_python_lib(1))")
+
+Name:           setools
+Version:        4.2.2
+Release:        2%{?setools_pre_ver:.%{setools_pre_ver}}%{?dist}
+Summary:        Policy analysis tools for SELinux
+# binaries are GPL and libraries are LGPL.  See COPYING.
+License:        GPLv2 and LGPLv2+
+URL:            https://github.com/SELinuxProject/setools
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://github.com/SELinuxProject/setools/releases/download/%{version}/%{name}-%{version}.tar.bz2
+
+BuildRequires: flex
+BuildRequires: bison
+BuildRequires: glibc-devel
+BuildRequires: gcc
+BuildRequires: git
+BuildRequires: libsepol-devel >= 2.9-1
+BuildRequires: qt5-qtbase-devel
+BuildRequires: swig
+BuildRequires: python3-Cython
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+BuildRequires: python3-xml
+BuildRequires: libselinux-devel
+
+%description
+SETools is a collection of graphical tools, command-line tools, and
+Python modules designed to facilitate SELinux policy analysis.
+
+%package     console
+Summary:     Policy analysis command-line tools for SELinux
+License:     GPLv2
+Requires:    setools-python3 = %{version}-%{release}
+Requires:    libselinux >= %{selinux_ver}
+
+%description console
+SETools is a collection of graphical tools, command-line tools, and
+libraries designed to facilitate SELinux policy analysis.
+
+This package includes the following console tools:
+
+  sediff       Compare two policies to find differences.
+  seinfo       List policy components.
+  sesearch     Search rules (allow, type_transition, etc.)
+
+%package     python3
+Summary:     Policy analysis tools for SELinux
+Obsoletes:   setools-libs < 4.0.0
+Recommends:  libselinux-python3
+Requires:    python3-setuptools
+
+%description python3
+SETools is a collection of graphical tools, command-line tools, and
+Python 3 modules designed to facilitate SELinux policy analysis.
+%prep
+%setup -n %{name}
+
+%build
+pwd
```

Comment:
What's the purpose of this `pwd` call?

Reply:
Was for debugging and I forgot to remove it.  I'll take it out, thanks.